NOnion/TorLogger.fs

```diff
@@ -9,6 +9,8 @@ module TorLogger =
         LoggerOpt <- Some loggingFunc
 
     let Log(msg: string) =
+        let msgWithDateTime = sprintf "[%s] %s" (DateTime.Now.ToString()) msg
```

Comment:
should it be UTC?

Reply:
I personally prefer this, but both are fine to me

Reply:
Changed to UTC